### PR TITLE
[stable/concourse] Concourse 5.3.0 Flags

### PR DESCRIPTION
**Pull request by taylorsilva**
#### What this PR does / why we need it:
Adds new flags introduced in concourse 5.3.0

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/concourse/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: concourse
-version: 6.2.1
+version: 6.3.0
 appVersion: 5.2.0
```
Comment (william-tran) [inline stable/concourse/Chart.yaml]: update this to 5.3.0

**Review by william-tran — CHANGES_REQUESTED** [review]

**Comment by aegershman** [comment]
Sidenote, sorry-- was the imageTag supposed to be updated to 5.3.0 in this PR?

the AppVersion was bumped to 5.3.0 (https://github.com/helm/charts/pull/14292/files#diff-459fb13dcf5560eb0c9f742a9d5109deR4)

https://github.com/helm/charts/blob/master/stable/concourse/values.yaml#L22

**Comment by cirocosta** [comment]
damnnn, we do that **every single time** hahah - I guess this is something that we should be paying more attention to ...

Internally, we're never really caught by this as we use digests, but this can be quite annoying for others 😞  

Thanks for spotting it, @aegershman !

**Comment by aegershman** [comment]
Will create a new issue for this if needed, but--

Is there an intended way to use this? Here's a trimmed down example:

```
concourse:
  web:
    externalUrl: "https://concourse.example.com"
    letsEncrypt:
      enabled: true

web:
  ingress:
    enabled: true
    hosts:
    - concourse.example.com
```

and pods fail to start due to the following message:

```
kubectl -n concourse-external logs -f concourse-external-web-84d86b5b55-bnpgq
1 error occurred:
        * must specify --tls-bind-port to use TLS
```

[but the only way to specify `tls-bind-port` is to turn on `concourse.web.tls.enabled=true`](https://github.com/helm/charts/blob/master/stable/concourse/templates/web-deployment.yaml#L178-L185), which will then require you to provide `secrets.webTlsCert` and `secrets.webTlsKey`, despite the fact that it's supposed to be auto-generated by certbot...?

Any thoughts?

**Comment by cirocosta** [comment]
hmmmm as far as I can tell, the letsencrypt integration would only work without the use of ingress as `web` would be the thing doing the TLS termination, instead of the ingress controller 🤔  

Anyway, I think it's good to have an issue so we can discuss and, in the "worst" case, come up with documentation that lets users know how to properly make use of it 😁  

Thanks again, @aegershman !

**Comment by aegershman** [comment]
Sure thing, I opened an issue about it https://github.com/helm/charts/issues/14867 I'll play around w/ it a bit more later

**Pull request merged**